lib/foreman_bootdisk/engine.rb

```diff
@@ -65,6 +55,81 @@ class Engine < ::Rails::Engine
           cx.add_pagelet :subnet_index_action_buttons, name: 'Bootdisk', partial: 'subnets/bootdisk_action_buttons'
           cx.add_pagelet :subnet_index_title_buttons, name: 'Bootdisk', partial: 'subnets/bootdisk_title_buttons'
         end
+
+        settings do
+          category :bootdisk, N_("Boot disk") do
+            ipxe = ['/usr/lib/ipxe'].find { |p| File.exist?(p) } || '/usr/share/ipxe'
+            isolinux = ['/usr/lib/ISOLINUX'].find { |p| File.exist?(p) } || '/usr/share/syslinux'
+            syslinux = ['/usr/lib/syslinux/modules/bios', '/usr/lib/syslinux'].find { |p| File.exist?(p) } || '/usr/share/syslinux'
+            grub2 = ['/var/lib/tftpboot/grub2'].find { |p| File.exist?(p) } || '/var/lib/foreman/bootdisk'
+            templates = -> {
+              Hash[ProvisioningTemplate.where(template_kind: TemplateKind.where(name: 'Bootdisk')).map { |temp| [temp[:name], temp[:name]] }]
+            }
+
+            setting "bootdisk_ipxe_dir",
+              type: :string,
+              default: ipxe,
+              full_name: N_("iPXE directory"),
+              description: N_("Path to directory containing iPXE images")
+
+            setting "bootdisk_isolinux_dir",
+              type: :string,
+              default: isolinux,
+              full_name: N_("ISOLINUX directory"),
+              description: N_("Path to directory containing isolinux images")
+
+            setting "bootdisk_syslinux_dir",
+              type: :string,
+              default: isolinux,
```

Comment:
I think Rubocop complains because of this oversight? :)
```suggestion
              default: syslinux,
```

Reply:
That bastard actually useful? Hmmm.